hibp.js

```diff
@@ -80,7 +82,7 @@ const HIBP = {
     } catch (error) {
       throw new Error("Could not load breaches: " + error);
     }
-    console.log("Done loading breaches.");
+    log.info("Done loading breaches");
```

Comment:
This may be the first instance of a sentence-like first param, where everything else was "loadBreachesIntoApp", "smtpUrl-empty", "removed-subscriber-not-found", "unhandled-eventType", etc.